style.css

```diff
@@ -0,0 +1,174 @@
+@import url('https://fonts.googleapis.com/css2?family=Oswald&display=swap');
```

Comment:
Please add the linters file to check errors in your codes
to add linter to your projects follow this link, click here https://github.com/microverseinc/linters-config